config/dev.exs

```diff
@@ -2,10 +2,11 @@ use Mix.Config
 
 # Configure your database
 config :wallet, Wallet.Repo,
-  username: "vinicius.arruda",
-  password: "monsterapi",
-  database: "wallet_dev",
-  hostname: "localhost",
+  username: "root",
+  password: "root",
+  database: "app_dev",
+  hostname: "app_mysql",
+  port: 3306,
```

Comment:
To work db, don't need add in dockerfile an container?

Reply:
No, the container is on docker-compose